mailbox/api/pom.xml

```diff
@@ -70,6 +70,15 @@
             <artifactId>junit-contracts</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-guava</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>commons-lang</groupId>
```

Comment:
Ok, there's a rule written nowhere about pom.xml.
We have a sorting algorithm : always sort alphabetically on groupId then artifactId and split internal dependencies from external ones (first we list all the org.apache.james dependencies, then all others)